Lib/test/test_clinic.py

```diff
@@ -1849,17 +1847,14 @@ def test_scaffolding(self):
         self.assertEqual(repr(clinic.NULL), '<Null>')
 
         # test that fail fails
-        expected = (
-            'Error in file "clown.txt" on line 69:\n'
-            'The igloos are melting!\n'
-        )
         with support.captured_stdout() as stdout:
             errmsg = 'The igloos are melting'
             with self.assertRaisesRegex(clinic.ClinicError, errmsg) as cm:
                 clinic.fail(errmsg, filename='clown.txt', line_number=69)
             exc = cm.exception
             self.assertEqual(exc.filename, 'clown.txt')
             self.assertEqual(exc.lineno, 69)
+            self.assertEqual(stdout.getvalue(), "")
```

Comment:
So the context manager is not needed either?

Reply:
I guess it's not _strictly_ needed, no. But I think it's maybe useful to assert that `clinic.fail()` never prints anything to stdout?

Reply:
Yeah, sounds good to me.